plugins/inputs/tail/tail.go

```diff
@@ -223,6 +226,21 @@ func (t *Tail) tailNewFiles(fromBeginning bool) error {
 				// we're already tailing this file
 				continue
 			}
+
+			//Check whether tailing has started
+			flag = true
```

Comment:
Also, no point of checking if file is tailed, as there is exactly the same check on line 225